### PR TITLE
Added a population dataset

### PR DESCRIPTION
**Pull request by jacgrout**
Added a dataset of ONS Population Estimates for 2023 for the UK broken down by country, age, and sex

Closes #95

### CODE REVIEW

**Comment by tomjemmett** [comment]
Can you run devtools::document(), that should resolve the error.

There's a note about the all contributors file @Lextuga007 this should be in .Rbuildignore

**Comment by Lextuga007** [comment]
I'm just fixing #86 😄

File: R/ons_uk_population_2023.R [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+#' @usage data(ons_uk_population_2023)
+#'
+#' @examples
+#' data(ons_uk_population_2023)
+#'
+#' library(readxl)
+#' library(tidyverse)
+#' library(tidyr)
+
+#' Load the data in
```
Comment (tomjemmett) [inline R/ons_uk_population_2023.R:36]:
```suggestion
#' # Load the data in
```

that line needs to be forced to be a comment: it's trying to run `Load the data in` as an R command

**Review by tomjemmett — CHANGES_REQUESTED** [review]

File: R/ons_uk_population_2023.R (hunk 1 above)
Comment (tomjemmett) [reply R/ons_uk_population_2023.R:36]: actually, I would remove and change the entire example here:

* the example is meant to be about using the data, not how to generate it
* the example includes a path to a file on your local computer

Comment (jacgrout) [reply R/ons_uk_population_2023.R:36]: Yeah, thats all been fixed just now anyway, but still failing !

**Review by tomjemmett — APPROVED** [review]

**Comment by Lextuga007** [comment]
@all-contributors please add @jacgrout for data 

Thank you for contributing such a major PR as part of #Hacktoberfest! 🎉

**Pull request merged**